### ISSUE TITLE
[New feature] Tag template

### ISSUE THREAD

**Issue by kikouk42** [ForkIssues/TrackerWin#644]
Hi Dan and Tanya !
Would it be possible in future release to add a tag template feature for annotated tag please ?

Currrently I use the following command : `tag -e -F <mytemplate_file> tag_name`

**Comment by DanPristupov** [comment]
Well, this is part of *your* workflow. It doesn't to fit to all users.

I think there's no way to achieve that until I implement the ability to prompt tag names for custom commands.

**Comment by kikouk42** [comment]
Yes, I agree but only partially.
I understand my current request seems very specific.
But It could be very helpful to have a user interface which give access to largest number of git commands.
Do you think it can suit several users need to implement such feature with a "browse" button to choose the file to load the annotated text field ?

**Comment by DanPristupov** [comment]
> Do you think it can suit several users need to implement such feature

Yes, most likely it could be helpful for some users. However I think that overall effect of keeping UI clean, minimalistic and not bloated is more important.
